code/modules/codex/categories/category_substances.dm

```diff
@@ -80,9 +73,19 @@
 			if(mat.ore_compresses_to && mat.ore_compresses_to != mat.type)
 				var/decl/material/M = decls_repository.get_decl(mat.ore_compresses_to)
 				material_info += "<li>It can be compressed into [M.solid_name].</li>"
-			if(mat.ore_smelts_to && mat.ore_smelts_to != mat.type)
-				var/decl/material/M = decls_repository.get_decl(mat.ore_smelts_to)
-				material_info += "<li>It can be smelted into [M.solid_name].</li>"
+			if(length(mat.heating_products))
+				var/list/heat_prod = list()
+				for(var/mtype in mat.heating_products)
+					var/decl/material/M = decls_repository.get_decl(mtype)
+					heat_prod += "[mat.heating_products[mtype] * 100]% [M.solid_name]"
+				material_info += "<li>It can be smelted into [english_list(heat_prod)] at a temperature of [mat.heating_point] Kelvin.</li>"
```

Comment:
woldn't this result in 'toxin can be smelted into 100% denaturated toxin'?

Reply:
Yes, but the whole file needs to be broken up to generate categories. I was thinking of doing that after this PR since it'll involve splitting the entire codex category up.